reconcile/utils/ocm/ocm.py

```diff
@@ -460,15 +463,15 @@ def update_machine_pool(self, cluster, spec):
         """
         cluster_id = self.cluster_ids[cluster]
         machine_pool_id = spec["id"]
-        labels = spec.get("labels", {})
+        labels: str = spec.get("labels", {})  # type: ignore
```

Comment:
I don't understand that. Why `str` but the default of the dict.get is `{}`?

Reply:
I don't know, but I don't have the courage to change it.

Reply:
Solved, I mixed the type annotation from 1 method to another 🤦